optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/es/ProcessInstanceRepositoryES.java

```diff
@@ -70,14 +70,19 @@ public void deleteByIds(
         BulkRequest.of(
             b ->
                 b.operations(
-                    o -> {
-                      processInstanceIds.forEach(
-                          id ->
-                              o.delete(
-                                  d ->
-                                      d.id(id).index(esClient.addPrefixesToIndices(index).get(0))));
-                      return o;
-                    }));
+                    processInstanceIds.stream()
```

Comment:
I am assuming this bug will also be present with OS, could you please do the equivalent change on ProcessInstanceRepositoryOS as well?